mkt/webapps/tasks.py

```diff
@@ -715,6 +715,10 @@ def _get_trending(app_id):
         return {}
 
     def _score(week1, week3):
+        # If last week app installs are < 100, this app isn't trending.
+        if week1 < 100.0:
```

Comment:
The idea is you don't want apps with a few installs to be considered as "trending". There's got to be some threshold.